api/src/main/java/com/bringback/yourlink/api/model/YourLink.java

```diff
@@ -0,0 +1,15 @@
+package com.bringback.yourlink.api.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class YourLink {
+    private String url;
+    private boolean isRed;
```

Comment:
**isRed** variable means isRead?

Reply:
Review notification test

Reply:
Review notification test2

Reply:
Oh :-) I will change to `isRead`.
```suggestion
    private boolean isRead;
```

Reply:
Oh :-) I will change to `read`
```suggestion
    private boolean read;
```